wgpu-core/src/binding_model.rs

```diff
@@ -167,6 +167,8 @@ pub enum CreateBindGroupError {
     MissingTextureUsage(#[from] MissingTextureUsageError),
     #[error("Binding declared as a single item, but bind group is using it as an array")]
     SingleBindingExpected,
+    #[error("Effective buffer binding size {size} is not a multiple of {alignment}")]
```

Comment:
suggestion: I'll try rephrasing this to indicate the expectation (adding that this is for storage buffers), and then noting the offending size.

Reply:
Tried this out with cebabaa6. Curious what the eventual reviewer for this thinks.